### ISSUE TITLE
[SLUS-21621] Persona 3 FES cutscene stutters on HDD

### ISSUE THREAD

**Issue by AmyRoxwell** [rickgaiser/neutrino#46]
Launching Persona 3 FES on neutrino on a HDD causes the cutscenes to suffer stutters, very noticeable on the first one after you had boot the game while using a HDD.

On OPL this can be fixed by enabling MODE 2 (Synchronous Mode) but enabling any of the settings on NHDDL do nothing.

Neutrino version used: neutrino-20241103-2ccca2d8
I'm also launching Neutrino via NHDDL.

**Comment by pcm720** [comment]
> is this possible in NHDDL? 

Yeah, compatibility flags can be toggled from the options menu for each ISO individually.
They're passed as `-gc=?` to Neutrino.

I also tried launching P3FES on my PS2 and had zero issues with FMVs, by the way.
But I'm using a microSD as my "HDD" via IDE2SD adapter, not an actual SSD or HDD

> PS: I added a link to your project from the readme.

Thanks!

**Comment by DukeJP** [comment]
I can't confirm the bug, the game works perfectly on HDD with the latest Neutrino and Nhddl (sync reads enabled). Could it be that you are using a clone adapter from Aliexpress? I had problems in GTA3 (the game freezes loading every few seconds) with a clone, but an original adapter worked perfectly.

**Comment by AmyRoxwell** [comment]
> I can't confirm the bug, the game works perfectly on HDD with the latest Neutrino and Nhddl (sync reads enabled). Could it be that you are using a clone adapter from Aliexpress? I had problems in GTA3 (the game freezes loading every few seconds) with a clone, but an original adapter worked perfectly.
> 
> EDIT: About Monster Hunter G, I tried the translated version (same ID) without problems. And Monster Hunter 1, at least the USA version works fine for me

I'm using the `Gamestar` adapter from amazon and a laptop HDD, idk why only neutrino gets funky with it and not OPL hmm

**Comment by rickgaiser** [comment]
Did you try mode 0, or mode 0 and 2 as I suggested?

**Comment by AmyRoxwell** [comment]
> Did you try mode 0, or mode 0 and 2 as I suggested?

Haven't been able to be at home, so not yet, I will try as soon as possible, sorry.

**Comment by DukeJP** [comment]
> 
> I'm using the `Gamestar` adapter from amazon and a laptop HDD, idk why only neutrino gets funky with it and not OPL hmm

afaik, it's not original then. The official Sony ones are for Ide disks (can be adapted to SATA). For me, a clone gave me the problems I mentioned above, even some OPL versions didn't even boot. As I said, the games you mention work perfectly with "sync reads" activated (in fact, it's a standard for me when testing games).
If you have everything installed and configured correctly and your game still stutters, the problem is your hardware.

**Comment by AmyRoxwell** [comment]
> Did you try mode 0, or mode 0 and 2 as I suggested?

I wonder, how can I do that without connecting to a network? the gamestar adapter does not have a network port

**Comment by pcm720** [comment]
> I wonder, how can I do that without connecting to a network? the gamestar adapter does not have a network port

You know you can just press triangle in NHDDL and toggle the modes you need, right?

**Comment by AmyRoxwell** [comment]
> > I wonder, how can I do that without connecting to a network? the gamestar adapter does not have a network port
> 
> You know you can just press triangle in NHDDL and toggle the modes you need, right?

Yes, but like I explained they don't do anything compared to OPL. Not matter the option I toggle it just doesn't do anything anything different.

**Comment by DukeJP** [comment]
As a test, I launched the game without any option activated and it also works fine, at least referring to the stutter of the intro

**Comment by AmyRoxwell** [comment]
Then probably it's an incompatibility with my HDD and/or Gamestar adapter, I would still need to test back home just in case.

**Comment by pcm720** [comment]
> Yes, but like I explained they don't do anything compared to OPL. Not matter the option I toggle it just doesn't do anything anything different.

What did you do after toggling the compatibility flags?
Did you launch the game with the square button straight from the options menu or save the options with the Start button?

**Comment by AmyRoxwell** [comment]
> > Yes, but like I explained they don't do anything compared to OPL. Not matter the option I toggle it just doesn't do anything anything different.
> 
> What did you do after toggling the compatibility flags? Did you launch the game with the square button straight from the options menu or save the options with the Start button?

I did the following:
1. Press triangle to show the option menu
2. Did the following combination of options:
A. Disable Compatibility Flags + Sync Mode
B. Disable Compatibility Flags + Accurate Reading 
C. Sync Mode Only
3. Press Start and Press triangle again to verify my settings, then start again
4. Run the game.

**Comment by pcm720** [comment]
Okay, that means Neutrino got the gc argument.

**Comment by 10522** [comment]
### Shin Megami Tensei Persona 3 FES [SLUS_216.21]:
NHDDL 1.2.1 + Neutrino 1.7.0-30 + udpbd-vexfat 0.2.0 - works well

My PS2 model is SCPH-90008. I use redump images and don't use PADEMU, VMC, GSM.

@rickgaiser @AmandaRoseChaqueta
